<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record</string>
  </property>
  <widget class="QPushButton" name="recordNStopButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop recording...</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time: 99:99</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileNameLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
